x-pack/plugins/monitoring/server/lib/create_query.ts

```diff
@@ -133,3 +133,84 @@ export function createQuery(options: {
 
   return query;
 }
+
```

Comment:
this is the same as `createQuery` but accepts a `productType` and gets rid of `type` and only uses `types`.  I didn't want to have to update all the code right away that doesn't need datastreams yet but uses the createQuery.